generation/readme_update.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# This script should run as a part of new-library generation process.
+# This script introduces release-please annotations if they don't exist in the readme file
+
+for module in $(find . -mindepth 2 -maxdepth 2 -name pom.xml | sort | xargs dirname); do
+
+  if [[ "${module}" = *java-core ]] || [[ "${module}" = *java-shared-dependencies ]] || [[ "${module}" = *java-iam ]]; then
```

Comment:
Should java-iam be excluded? If so, can you add source code comment why?

Reply:
I don't think it should have been excluded. I've added it back again, along with necessary changes in its readme.